bugsnag-plugin-android-ndk/src/main/jni/bugsnag_ndk.h

```diff
@@ -57,10 +58,14 @@ typedef struct {
      * true if a handler has completed crash handling
      */
     bool crash_handled;
+
+    bsg_on_error on_error;
 } bsg_environment;
 
 bsg_unwinder bsg_configured_unwind_style();
 
+bool bsg_run_on_error();
```

Comment:
Could use some docs here to say what this does, and what the return value means. There might also be a more descriptive name, since its always used as a condition to persist reports? "should save report", "update and filter report" maybe?